frontend/packages/console-shared/src/components/error/fallbacks/ErrorBoundaryFallbackPage.tsx

```diff
@@ -1,26 +1,42 @@
 import * as React from 'react';
+import { ErrorState } from '@patternfly/react-component-groups';
+import {
+  ActionList,
+  ActionListGroup,
+  ActionListItem,
+  Button,
+  ButtonVariant,
+} from '@patternfly/react-core';
 import { useTranslation } from 'react-i18next';
 import { ErrorBoundaryFallbackProps } from '@console/dynamic-plugin-sdk';
-import { ExpandCollapse } from '@console/internal/components/utils/expand-collapse';
-import PaneBody from '@console/shared/src/components/layout/PaneBody';
-import PrimaryHeading from '../../heading/PrimaryHeading';
-import ErrorDetailsBlock from './ErrorDetailsBlock';
+import { ErrorDetailsModal } from './ErrorDetails';
 
 /**
  * Standard fallback catch -- expected to take up the whole page.
  */
 const ErrorBoundaryFallbackPage: React.FC<ErrorBoundaryFallbackProps> = (props) => {
   const { t } = useTranslation();
   return (
-    <PaneBody>
-      <PrimaryHeading centerText>{t('console-shared~Oh no! Something went wrong.')}</PrimaryHeading>
-      <ExpandCollapse
-        textCollapsed={t('console-shared~Show details')}
-        textExpanded={t('console-shared~Hide details')}
-      >
-        <ErrorDetailsBlock {...props} />
-      </ExpandCollapse>
-    </PaneBody>
+    <ErrorState
+      titleText={t('console-shared~Oh no! Something went wrong.')}
```

Comment:
Per https://www.patternfly.org/ux-writing/error-messages/#best-practices-for-404-error-content, am thinking this needs updating as well.  Maybe
```suggestion
      titleText={t('console-shared~An error occurred')}
```

Reply:
I'll change it to "Something wrong happened" to match PF's `ErrorBoundary`: https://www.patternfly.org/component-groups/error-communication/error-boundary